### PR TITLE
Admin views and order show page#36

### LINKED ISSUE #36: User Story 34, Admin views a User's Order Show Page
```
As an admin user
When I visit a user's profile
And I click on a link for order's show page
My URL route is now something like "/admin/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item the user ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #38

depends on #49

### PR DESCRIPTION
**Pull request by danbriechle**
Pulled code from PR #169 
Adds test to for Admin sees user order show page

test passing no additional code needed. 

closes #36

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Admin::OrdersController < ApplicationController
+  before_action :require_admin
+
+def show
+  @order = Order.find(params[:id])
+end
+
+def update
+  order = Order.find(params[:id])
+    order.order_items.each do |item|
```
Comment (asmolentzov) [inline app/controllers/admin/orders_controller.rb]: same comment here as on PR 34 regarding indentation :)
